test/middleware-test.ts

```diff
@@ -0,0 +1,710 @@
+// /test/middleware-test.ts
+// Tests the rate limiting middleware
+
+import { jest } from '@jest/globals'
+import Express from 'express'
+import Sinon, { SinonFakeTimers } from 'sinon'
```

Comment:
We should be able to replace this with the Jest equivalents - https://jestjs.io/docs/jest-object#mock-timers 

Although, I suppose if this working, maybe we should just leave it for now.

Reply:
I just changed that as well as part of https://github.com/nfriedly/express-rate-limit/pull/263#discussion_r774300344